### DISCUSSION TITLE
Are `Actor` based classes recognized by the engine in UI?

### DISCUSSION THREAD

**Discussion by prokopst** [nxrighthere/UnrealCLR#51] [Help] [answered]
I wanted to interact with a custom `Actor` (and a custom `SceneComponent`) created using UnrealCLR but the engine doesn't see them. Are `Actor` based classes actually recognized by the engine in UI? Is it the same problem as exposing that to Blueprints, #33 ?

**Comment by nxrighthere** [comment, marked as answer]
Nope, managed classes are not visible to the engine (see https://github.com/nxrighthere/UnrealCLR/issues/45 and https://github.com/nxrighthere/UnrealCLR/discussions/46).

**Reply by prokopst** [reply]
I see. I think it would be helpful for people coming to the project to understand what's implemented by the plugin, what hasn't been implemented yet and what's out of scope. The `README.md` seems to list only features.

**Reply by nxrighthere** [reply]
Yes, because it's still work in progress and things might change before the release.
